src/v2/Apps/Artwork/Components/ArtworkMeta.tsx

```diff
@@ -111,6 +125,14 @@ export class ArtworkMeta extends Component<ArtworkMetaProps> {
     )
   }
 
+  renderZendeskScript() {
+    const { artwork } = this.props
+    if (artwork.is_in_auction || !artwork.is_acquireable) return
```

Comment:
_I think_ `is_acquireable` only includes Buy Now works and looks like we want to enable Zendesk for Make Offer and inquiry works.

Reply:
For the requirement of only showing the chat for artworks over certain price point, instead of doing the currency conversion ourselves, I wonder if we can just get the fixed threshold for each currency we support.

Reply:
Do you think this would need to be some sort of field that lives in metaphysics?

Reply:
I was thinking we can have the fixed threshold for each currency set in Force directly to begin with, but escalating this concept to MP might make sense in the future too.